source/administration/object-management/transition-objects-to-s3.rst

```diff
@@ -189,11 +194,18 @@ The example above uses the following arguments:
        diagnostics, maintenance, or disaster recovery.
 
    * - :mc-cmd:`STORAGE_CLASS <mc ilm tier add --storage-class>`
-     - The S3 storage class to which MinIO transitions objects. Specify
-       one of the following supported storage classes:
+     - The S3 storage class to which MinIO transitions objects. 
+       MinIO *requires* the remote storage class implement immediate object retrieval with no rehydration or manual intervention required.
```

Comment:
As on the other two pages regarding `implement`.

Reply:
Rewrote